src/refiners/foundationals/swin/mvanet/converter.py

```diff
@@ -0,0 +1,138 @@
+import re
+
+from torch import Tensor
+
+
+def convert_weights(official_state_dict: dict[str, Tensor]) -> dict[str, Tensor]:
+    rm_list = [
+        # Official weights contains useless keys
+        # See https://github.com/qianyu-dlut/MVANet/issues/3#issuecomment-2105650425
+        r"multifieldcrossatt.linear[56]",
+        r"multifieldcrossatt.attention.5",
+        r"dec_blk\d+\.linear[12]",
+        r"dec_blk[1234]\.attention\.[4567]",
+        # We don't need the sideout weights
```

Comment:
I guess this is "... at inference-time"

Reply:
Yes, IMO if we add things just for training it must be in training-specific code (using adapters etc).

Reply:
Worth expanding to "We don't need the sideout weights at inference-time" then (just to make it 100% clear)?